src/project/internal/projectconfiguration.cpp

```diff
@@ -305,41 +228,52 @@ void ProjectConfiguration::setShouldAskSaveLocationType(bool shouldAsk)
     settings()->setSharedValue(SHOULD_ASK_SAVE_LOCATION_TYPE, Val(shouldAsk));
 }
 
-bool ProjectConfiguration::isCloudProject(const io::path_t& projectPath) const
+io::path_t ProjectConfiguration::oldCloudProjectsPath() const
 {
-    return io::dirpath(projectPath) == cloudProjectsPath();
+    return globalConfiguration()->userDataPath() + "/Cloud Scores";
 }
 
-io::path_t ProjectConfiguration::cloudProjectSavingFilePath(const io::path_t& projectName) const
+io::path_t ProjectConfiguration::newCloudProjectsPath() const
```

Comment:
is it a temporary solution?
If not, then please explain how it works

Reply:
It is temporary. As mentioned in the PR description, we should put cloud scores in a hidden location; that is the `newCloudProjectsPath()`. But we also want that existing cloud scores are still treated as cloud scores. Maybe we should add something about compatibility, to migrate "old" cloud scores to the "new" cloud folder. Or we should do this migration at the moment that the user saves such an old cloud score, and convert it into a new one at that moment.